### PR TITLE
WEBUI-1260: backward port: need to downgrade chrome browser version

### PR DESCRIPTION
**Pull request by rahuljain-dev**
https://jira.nuxeo.com/browse/WEBUI-1260

note for a11y changes:
Version[axe-core]:4.7.2 
[home.js] color-contrast changed to color-contrast-enhanced due to version upgrade to 4.8.1

### CODE REVIEW

**Review by rakeshkumar1019 — APPROVED** [review]
Good Job 👍.

File: plugin/a11y/test/specs/home.js [hunk 1]
```diff
@@ -18,7 +17,7 @@ const EXPECTED_VIOLATIONS = {
 
 const EXPECTED_INCOMPLETE_VIOLATIONS = {
   'aria-allowed-role': 5,
-  'color-contrast': 8,
+  'color-contrast-enhanced': 2,
```
Comment (poonamyadav252) [inline plugin/a11y/test/specs/home.js:20]: This violation should be resolved, Please check if possible to resolve this.

**Review by poonamyadav252 — CHANGES_REQUESTED** [review]

File: plugin/a11y/test/specs/home.js (hunk 1 above)
Comment (rahuljain-dev) [reply plugin/a11y/test/specs/home.js:20]: This change is done as we move from axe core version from 4.7.2 to 4.8.1.

a11y report with axe-core version 4.7.2:

incomplete: [
[INFO]      [exec] [0-1]     { id: 'aria-allowed-role', issues: 5 },
[INFO]      [exec] [0-1]     { id: 'aria-required-children', issues: 1 },
[INFO]      [exec] [0-1]     { id: 'color-contrast', issues: 2 }
[INFO]      [exec] [0-1]   ],
[INFO]      [exec] [0-1]   violations: [
[INFO]      [exec] [0-1]     { id: 'aria-allowed-attr', issues: 4 },
[INFO]      [exec] [0-1]     { id: 'aria-command-name', issues: 1 },
[INFO]      [exec] [0-1]     { id: 'aria-required-children', issues: 10 },
[INFO]      [exec] [0-1]     { id: 'aria-tooltip-name', issues: 1 },
[INFO]      [exec] [0-1]     { id: 'color-contrast', issues: 12 },
[INFO]      [exec] [0-1]     { id: 'duplicate-id', issues: 28 },
[INFO]      [exec] [0-1]     { id: 'landmark-one-main', issues: 1 },
[INFO]      [exec] [0-1]     { id: 'meta-viewport', issues: 1 },
[INFO]      [exec] [0-1]     { id: 'nested-interactive', issues: 15 },
[INFO]      [exec] [0-1]     { id: 'page-has-heading-one', issues: 1 },
[INFO]      [exec] [0-1]     { id: 'region', issues: 22 }
[INFO]      [exec] [0-1]   ]

a11y report with axe-core version 4.8.1:

[INFO]      [exec] [0-0]     { id: 'aria-allowed-attr', issues: 1 },
[INFO]      [exec] [0-0]     { id: 'aria-allowed-role', issues: 7 },
[INFO]      [exec] [0-0]     { id: 'aria-valid-attr-value', issues: 1 },
[INFO]      [exec] [0-0]     { id: 'color-contrast-enhanced', issues: 2 }
[INFO]      [exec] [0-0]   ],
[INFO]      [exec] [0-0]   violations: [
[INFO]      [exec] [0-0]     { id: 'aria-allowed-attr', issues: 1 },
[INFO]      [exec] [0-0]     { id: 'aria-command-name', issues: 1 },
[INFO]      [exec] [0-0]     { id: 'aria-required-children', issues: 4 },
[INFO]      [exec] [0-0]     { id: 'aria-roles', issues: 3 },
[INFO]      [exec] [0-0]     { id: 'aria-tooltip-name', issues: 1 },
[INFO]      [exec] [0-0]     { id: 'color-contrast-enhanced', issues: 15 },
[INFO]      [exec] [0-0]     { id: 'color-contrast', issues: 1 },
[INFO]      [exec] [0-0]     { id: 'duplicate-id', issues: 28 },
[INFO]      [exec] [0-0]     { id: 'landmark-one-main', issues: 1 },
[INFO]      [exec] [0-0]     { id: 'meta-viewport', issues: 1 },
[INFO]      [exec] [0-0]     { id: 'nested-interactive', issues: 15 },
[INFO]      [exec] [0-0]     { id: 'page-has-heading-one', issues: 1 },
[INFO]      [exec] [0-0]     { id: 'region', issues: 24 }
[INFO]      [exec] [0-0]   ]

This violation changed from color-contrast : 2 to color-contrast-enhanced : 2 as we moved to newer version but not due to any code changes.

**Review by rakeshkumar1019 — APPROVED** [review]

**Review by poonamyadav252 — APPROVED** [review]
Looks good, but only `violation` point needs to be discussed with the PO.

**Review by arupmmi07 — APPROVED** [review]
Reviewed

**Pull request merged**